core/runtime/src/main/java/io/quarkus/runtime/TemplateHtmlBuilder.java

```diff
@@ -12,17 +12,28 @@
             "</head>\n" +
             "<body>\n";
 
-    private static final String HTML_TEMPLATE_END = "</div></body>\n" +
+    private static final String HTML_TEMPLATE_END = "</body>\n" +
```

Comment:
Again, looks unrelated. Moreover, I think it's wrong because the header explicitly open a div for the content of the page.

Can you get rid of it?